data/json/items/armor/ammo_pouch.json

```diff
@@ -659,6 +711,138 @@
     "flags": [ "WATER_FRIENDLY", "BELTED" ],
     "armor": [ { "encumbrance": 2, "coverage": 85, "covers": [ "torso" ], "specifically_covers": [ "torso_upper" ] } ]
   },
+  {
+    "id": "tacvest_sling",
```

Comment:
What would you route the sling through, on this vest? Slings are usually routed through shoulder straps, like on [this plate carrier](https://cdn11.bigcommerce.com/s-l6o6hq0/images/stencil/original/products/160/1011/_DSC0028_edited_640__28208.1425071713.1280.1280_clipped_rev_1__42099.1710653842.jpg?c=2&imbypass=on&imbypass=on), or attached to the rig harness in other ways. I'm unable to find any examples of the real-life counterpart being bound with a sling.

Reply:
The vest has a D-ring on each shoulder

Reply:
![image](https://github.com/user-attachments/assets/0ef79dc6-b58b-4cfd-9d1b-9106054ee431)
I'm honestly not qualified to say whether these would be sufficient for holding a rifle or not, so if you or someone else more qualified than me has input, I'm all ears.

Reply:
I haven't noticed those. Might work! I can't say one way or another. Just going to reiterate that I haven't seen one image of those rings being utilized for anything.

Reply:
I'll be honest, it's been difficult finding *any* images of these vests actually being worn by anyone. If no one objects I'll leave it, because I'm not really sure what else those would be used for. I can't really imagine using those to hang any other gear off of, but I'm really not familiar with the military world, or tactical gear.